src/main/java/view/VoiceSettingsView.java

```diff
@@ -0,0 +1,224 @@
+        labelTable.put(0, new JLabel("Female"));
+        labelTable.put(1, new JLabel("Male"));
+
+        speedSlider = new JSlider(JSlider.HORIZONTAL, 1, 10, SPEED_VALUE);
+        volumeSlider = new JSlider(JSlider.HORIZONTAL, 1, 10, VOLUME_VALUE);
+        voiceTypeSlider = new JSlider(JSlider.HORIZONTAL, 0, 1, VOICE_TYPE_VALUE);
+        pitchSlider = new JSlider(JSlider.HORIZONTAL, 1, 10, PITCH_VALUE);
+
+        speedLabel = new JLabel("Voice Speed: " + speedSlider.getValue());
+        volumeLabel = new JLabel("Voice Volume: " + volumeSlider.getValue());
+        voiceTypeLabel = new JLabel("Voice Type: " + getVoiceType(voiceTypeSlider.getValue()));
+        pitchLabel = new JLabel("Voice Pitch: " + pitchSlider.getValue());
+
+        speedSlider.setPaintTicks(true);
+        volumeSlider.setPaintTicks(true);
+        voiceTypeSlider.setPaintTicks(true);
+        pitchSlider.setPaintTicks(true);
+
+        voiceTypeSlider.setLabelTable(labelTable);
+
+        speedSlider.setPaintLabels(true);
+        volumeSlider.setPaintLabels(true);
+        voiceTypeSlider.setPaintLabels(true);
+        pitchSlider.setPaintLabels(true);
+
+        speedSlider.setMajorTickSpacing(1);
+        volumeSlider.setMajorTickSpacing(1);
+        voiceTypeSlider.setMajorTickSpacing(1);
+        pitchSlider.setMajorTickSpacing(1);
+
+        voiceTypeSlider.setMinorTickSpacing(1);
+        voiceTypeSlider.setSnapToTicks(true);
+
+        speedSlider.addChangeListener(e -> {speedLabel.setText("Voice Speed: " +
+                speedSlider.getValue());});
+        volumeSlider.addChangeListener(e -> {volumeLabel.setText("Voice Volume: " +
+                volumeSlider.getValue());});
+        voiceTypeSlider.addChangeListener(e -> {voiceTypeLabel.setText("Voice Type: " +
+                getVoiceType(voiceTypeSlider.getValue()));});
+        pitchSlider.addChangeListener(e -> {pitchLabel.setText("Voice Pitch: " +
+                pitchSlider.getValue());});
+
+        speedSlider.setFont(tickFont);
+        volumeSlider.setFont(tickFont);
+        voiceTypeSlider.setFont(tickFont);
+        pitchSlider.setFont(tickFont);
+
+        speedLabel.setFont(labelFont);
+        volumeLabel.setFont(labelFont);
+        voiceTypeLabel.setFont(labelFont);
+        pitchLabel.setFont(labelFont);
+
+        sliderPanel.add(voiceTypeLabel);
+        sliderPanel.add(voiceTypeSlider);
+        sliderPanel.add(speedLabel);
+        sliderPanel.add(speedSlider);
+        sliderPanel.add(volumeLabel);
+        sliderPanel.add(volumeSlider);
+        sliderPanel.add(pitchLabel);
+        sliderPanel.add(pitchSlider);
+
+        return sliderPanel;
+    }
+
+    private JPanel createButtonPanel() {
+        JPanel buttonPanel = new JPanel(new GridLayout(1, 1, 0, 0));
+        buttonPanel.setBackground(BACKGROUND_COLOR);
+
+        setButton = new GlowButton("Save Settings", PRIMARY_COLOR);
+        setButton.addActionListener(e -> customizeVoice());
+
```

Comment:
Maybe add a frame.setvisible(false) here to make the settings view disappear once the settings are saved?